package.json

```diff
@@ -429,6 +429,7 @@
     "test": "yarn compile",
     "build": "vsce package --yarn",
     "vscode:publish": "vsce publish --yarn",
+    "ovsx:publish": "ovsx publish",
```

Comment:
You could add the `--yarn` option here, too.

Reply:
I am reusing the previously build package and running publish on a file:

`yarn ovsx:publish ./metals-${{ steps.get_version.outputs.NEW_VERSION }}.vsix`

Would that also benefit from `--yarn`? Or since that file is already built it doesn't matter?

Reply:
Right, in that case it doesn't matter.

Reply:
Ok, thanks!